pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -60,25 +63,45 @@ var (
 	errInconsistentPodSetAssignments = errors.New("inconsistent podSet assignments")
 )
 
-var (
-	MaxRetries        int32 = 3
-	MinBackoffSeconds int32 = 60
-)
-
 type provisioningConfigHelper = admissioncheck.ConfigHelper[*kueue.ProvisioningRequestConfig, kueue.ProvisioningRequestConfig]
 
 func newProvisioningConfigHelper(c client.Client) (*provisioningConfigHelper, error) {
 	return admissioncheck.NewConfigHelper[*kueue.ProvisioningRequestConfig](c)
 }
 
+type waitForProvisioningRequestReadyConfig struct {
```

Comment:
A bit too much indirection. Just put these values inside the `options` struct.

Reply:
Thanks. Done.